.github/workflows/docker-image.yml

```diff
@@ -43,3 +43,11 @@ jobs:
         docker login https://docker.pkg.github.com -u qdrant --password ${{ secrets.GITHUB_TOKEN }}
         docker tag $DOCKERHUB_TAG $GITHUB_TAG
         docker push $GITHUB_TAG
+        
+        DOCKERHUB_TAG_UNPRIVILEGED="qdrant/qdrant:${{ github.ref_name }}-unprivileged"
+        TAGS_UNPRIVILEGED="-t ${DOCKERHUB_TAG_UNPRIVILEGED}"
+        GITHUB_TAG_UNPRIVILEGED="docker.pkg.github.com/qdrant/qdrant/qdrant:${{ github.ref_name }}-unprivileged"
+      
+        docker buildx build --build-arg='USER_UID=1000' --build-arg='USER_GID=1000' --build-arg='USERNAME=qdrant' --platform='linux/amd64,linux/arm64' $TAGS_UNPRIVILEGED --push --label "org.opencontainers.image.version"=$RELEASE_VERSION .
+        docker tag $DOCKERHUB_TAG_UNPRIVILEGED $GITHUB_TAG_UNPRIVILEGED
+        docker push $GITHUB_TAG_UNPRIVILEGED
```

Comment:
Should we also tag `:latest-unprivileged`?

Reply:
I don't need it for my use-case, but probably a good idea to stay consistent. @generall Thoughts?

Reply:
I added `:latest-unprivileged`